docs/features/purchase_management.md

```diff
@@ -0,0 +1,11 @@
+# Purchase Management Interface
+
+Invenio ILS simplifies the acquisition process, providing a comprehensive purchase management interface.
+
+## Order Management
+
+Libraries can efficiently manage their purchase orders, track expenditures, and maintain records of all acquisitions. This ensures that budgeting and procurement processes are smooth and well-documented.
```

Comment:
```suggestion
Libraries can efficiently manage their purchase orders, and maintain records of all acquisitions. This ensures procurement processes are smooth and well-documented.
```
removed these words, because it implied we had also invoice support which we don't have